src/wui/http/http_client.c

```diff
@@ -521,24 +529,19 @@ err_t data_received_fun(void *arg, struct tcp_pcb *tpcb, struct pbuf *p, err_t e
         return ERR_ARG;
     }
 
-    if (header_info.content_type == TYPE_INVALID){
-        cmd_status = CMD_REJT_CONT_TYPE;
+    if (false == header_info.valid_request) {
         pbuf_free(p);
+        return ERR_OK;
```

Comment:
We won't support BAD_REQUEST responses? I thought, that is why we have all the cmd_status_strs about header info

Reply:
again according to Ondra our firmware responds unnecessarily to his server responses. rejected message is needed on very specific and limited cases, mainly if command was rejected due to buffer full, etc